src/Jackett.Common/Indexers/MejorTorrent.cs

```diff
@@ -156,7 +156,7 @@ interface IScraper<T>
 
         class RssScraper : IScraper<IEnumerable<KeyValuePair<MTReleaseInfo, Uri>>>
         {
-            private readonly string LinkQuerySelector = "a[href*=\"/serie\"]";
+            private readonly string LinkQuerySelector = "a[href*=series_extend]";
```

Comment:
can you provide a link to a page that has this href, I don't seem to be able to find it.

Reply:
Yes, of course. 

On http://www.mejortorrentt.org/secciones.php?sec=ultimos_torrents which is used for last releases, on TV series releases links changed from /serie...... to:

"<a href="secciones.php?sec=descargas&amp;ap=series_extend&amp;id1=59534&amp;id2=59535&amp;titulo=NCIS-New-Orleans-5-Temporada" style="font-size:11px;">NCIS: New Orleans - 5ª Temporada: 5x23 -</a>"

In order to allow the indexer find new releases is needed to modify the query.
There is also a problem with the indexer searching tv series by season and episode that I expect to fix as soon as posible. Working on